exporter/tanzuobservabilityexporter/README.md

```diff
@@ -47,13 +55,19 @@ exporters:
   tanzuobservability:
```

Comment:
When Travis and I pair on updating this PR, let's consider promoting the Configuration section towards to the top of the README, since that's likely what users care about most to get the exporter running.